examples/go-client/service/service-poc.go

```diff
@@ -0,0 +1,272 @@
+// +build sample_client
+
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Comcast/rulio/core"
+	"github.com/Comcast/rulio/sys"
+	"github.com/tidwall/pretty"
+	"strings"
+	"time"
+)
+
+type Service struct {
```

Comment:
How is this `Service` different from `service.Service`? A comment there would be helpful.